### PR TITLE
Support `CredentialConfig.TLSSecret` with namespace. Resolve: #1066

### PR DESCRIPTION
**Pull request by zzjin**
Users can get/set https certs like: `kube-system/tls-secret`.

Full example:
```yaml
automaticHttps: false
fallbackForInvalidSecret: true
credentialConfig:
  - domains:
      - '*.sealos.io'
      - 'sealos.io'
    tlsSecret: sealos-system/wildcard-cert
```

fixes: #1066 

Note: **behaviour changed**.

Since add support of namespace/secretName, the `SecretNamePrefix = "higress-secret-"` prefix is no needed anymore, so I deleted it.

**This will re-issue automatic-managed certs** after upgrade to new version with `SecretMgr` cannot found cert secret without name prefix.

### CODE REVIEW

**Comment by johnlanni** [comment]
@zzjin Please sign the CLA.

**Comment by johnlanni** [comment]
cc @2456868764

File: pkg/cert/secret.go [hunk 1]
```diff
@@ -46,13 +41,20 @@ func NewSecretMgr(namespace string, client kubernetes.Interface) (*SecretMgr, er
...
 func (s *SecretMgr) Update(domain string, secretName string, privateKey []byte, certificate []byte, notBefore time.Time, notAfter time.Time, isRenew bool) error {
-	//secretName := s.getSecretName(domain)
-	secret := s.constructSecret(domain, privateKey, certificate, notBefore, notAfter, isRenew)
-	_, err := s.client.CoreV1().Secrets(s.namespace).Get(context.Background(), secretName, metav1.GetOptions{})
+	name := secretName
+	namespace := s.namespace
+	namespaceP, secretP := ParseTLSSecret(secretName)
+	if namespaceP != "" {
+		namespace = namespaceP
+		secretName = secretP
```
Comment (2456868764) [inline pkg/cert/secret.go:49]: 这里要把 secretP 赋值给 name

Comment (zzjin) [reply pkg/cert/secret.go:49]: Typo error. Fixed. THX.

**Comment by 2456868764** [comment]
LGTM. THX.

**Pull request merged**